pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -23,13 +23,17 @@ import (
 	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
 )
 
+// validateBindingName is the validation function for Binding names.
+var validateBindingName = apivalidation.NameIsDNSSubdomain
```

Comment:
why put this into a package-level var? it it used anywhere else besides the below?